bootstrap-rowequalizer.js

```diff
@@ -0,0 +1,120 @@
+/* global window, module, define, jQuery, require */
+/*
+ * Bootstrap RowEqualizer v1.0.0
+ * https://github.com/gsmke/bootstrap-rowequalizer
+ */
+
+;(function(factory) {
+    'use strict';
+
+    if (typeof define === 'function' && define.amd) {
+        define(['jquery'], factory);
+    } else if (typeof exports !== 'undefined') {
+        module.exports = factory(require('jquery'));
+    } else {
+        factory(jQuery);
+    }
+
+}(function($) {
+    'use strict';
+
+    var RowEqualizer = window.RowEqualizer || {};
+
+    RowEqualizer = (function () {
+        function RowEqualizer(element, settings) {
+
+            var _ = this;
+
+            _.el = element;
+
+            if (settings) {
+                _.totalColumns = settings;
+            } else {
+                _.totalColumns = 12;
+            }
+
+            _.colQueries = [
+                'xs',
+                'sm',
+                'md',
+                'lg'
+            ];
+            _.colQueryExist = {
+                xs: false,
+                sm: false,
+                md: false,
+                lg: false
+            };
+
+            for (var x = 0; x < _.colQueries.length; x++) {
+                if ($('> [class*=col-'+ _.colQueries[x] +']',_.el).length) {
+                    _.colQueryExist[_.colQueries[x]] = true;
+                }
+            }
```

Comment:
Checks which breakpoints are present in any of column classes and sets them to `true` in the object `colQueryExist`.